### DISCUSSION TITLE
Add "payment" sub-resource to incoming payment

### DISCUSSION THREAD

**Discussion by adrianhopebailie** [interledger/open-payments#414] [Ideas]
It would be very useful to send information from the sending ASE to the receiving ASE after each payment toward an incoming payment, especially if the incoming payment has no amount (e.g. Web Monetization).

## Proposal

The sending wallet can POST a new "payment" resource to the incoming-payment which adds a sub-resource that represents some money was sent to the incoming payment.

Possible properties of the resource:
 - amount sent
 - sender
 - meta

**Comment by mkurapov** [comment]
I'm guessing this would be an attribute that is a _list_ of "payments"? Since we can have multiple outgoing payments per incoming payment

(Dare I suggest we call this "receipts"...)
